### PR TITLE
Fix /appeal language

### PR DESCRIPTION
**Pull request by eudaldca**

### CODE REVIEW

File: app/views/appeals/appeal.haml [hunk 1]
```diff
@@ -10,16 +10,16 @@
...
             %h4.strong What can be appealed?
-            %p If you have been caught cheating, your appeal will be denied.
-            %p Appeals exist in order to catch mistakes, not to argue about the rules, or your personal virtues.
+            %p Any punishment which you believe was issued incorrectly or which is too severe can be contested.
+            %p Appeals exist in order to catch mistakes, not to argue about the rules.
         .col-md-4
             %h4.strong How are appeals resolved?
-            %p The moderator who punished you will review the evidence, and remove the punishment if they find it was in error.
-            %p You can escalate an appeal to have it reviewed by a senior staff member, but this will delay the resolution.
+            %p The moderator who punished you will review the incident, and appeal the punishment if they find it was in error or reduce it in severity if they feel it was too harsh.
```
Comment (AustinLMayes) [inline app/views/appeals/appeal.haml]: should be `either appeal, or reduce the severity of, the punishment if they find it was in error or too harsh`

**Comment by AustinLMayes** [comment]
`Infraction` and `Punishment` are both used, we should stick to one or the other

File: app/views/appeals/appeal.haml (hunk 1 above)
Comment (eudaldca) [reply app/views/appeals/appeal.haml]: No

File: app/views/appeals/appeal.haml [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
-            %p If you have been caught cheating, your appeal will be denied.
-            %p Appeals exist in order to catch mistakes, not to argue about the rules, or your personal virtues.
+            %p Any infraction which you believe was issued incorrectly or which is too severe can be contested.
+            %p Appeals exist in order to catch mistakes, not to argue about the rules.
         .col-md-4
             %h4.strong How are appeals resolved?
-            %p The moderator who punished you will review the evidence, and remove the punishment if they find it was in error.
-            %p You can escalate an appeal to have it reviewed by a senior staff member, but this will delay the resolution.
+            %p The moderator who punished you will review the incident, and appeal the infraction if they find it was in error or reduce it in severity if they feel it was too harsh.
+            %p If you disagree with the moderator’s decision, you can escalate an appeal to have it reviewed by a senior staff member, but this will delay the resolution.
```
Comment (potatostealer) [inline app/views/appeals/appeal.haml]: "Delay the resolution" should be "Delay the response time"

File: app/views/appeals/appeal.haml [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
         .col-md-4
-            %h4.strong Privilege, not a right
-            %p Playing on our servers is a privilege.
-            %p If you abused or have shown promise to abuse the network, your appeal will be denied.
+            %h4.strong Conduct in appeals
+            %p If your infraction was issued incorrectly, you should appeal it in a timely manner. Moderators are only required to obtain evidence for cheating bans, and this must only be retained for 28 days.
+            %p If you lie in your appeal or are abusive towards our staff, your appeal will be denied and you may receive further punishment.
```
Comment (potatostealer) [inline app/views/appeals/appeal.haml:22]: "You may receive further punishment" should be "harsher punishments may be issued"

**Review by HuManKeat — CHANGES_REQUESTED** [review]

**Comment by eudaldca** [comment]
All fixed

**Pull request merged**